package.json

```diff
@@ -75,7 +75,9 @@
     "remove-markdown": "^0.3.0",
     "shortid": "^2.2.16",
     "slugify": "^1.4.5",
+    "styled-components": "^5.2.0",
```

Comment:
do we need that? That's a big dependency

Reply:
noo, forgot it

Reply:
it's one of the weird dependencies required when i use react hooks locally